lib/modules/manager/npm/post-update/__snapshots__/pnpm.spec.ts.snap

```diff
@@ -1,6 +1,8 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`modules/manager/npm/post-update/pnpm catches errors 1`] = `
+exports[`modules/manager/npm/post-update/pnpm catches errors 1`] = `Array []`;
```

Comment:
Please don't use snapshot expectation for empty or small objects. Use other inline matchers instead.

Reply:
i dont remmember changing this. i will check the tests O.o

Reply:
![image](https://user-images.githubusercontent.com/42116482/161734049-39cb4a9f-4c39-4f92-be61-454c25b4c5a4.png)

according to docs, its a field that should be on a project level, not workspace

Reply:
fixed test